### PR TITLE
Added product update for Single Change + visual

### PR DESCRIPTION
**Pull request by jayk-u**
Here's my draft about the Product Update Single Change

Please let me know if anything needs to be improved (I plan to post this one on next Tuesday, can be adjusted)

### CODE REVIEW

File: src/_product-updates/2024-05-21-single-change.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Single Change
+tags: [New]
+image: /images/changelog/single-change.png
+---
+
+![single-change.png](/images/changelog/single-change.png)
+
+The changelog allows the consumers of your API documentation to track changes as your documentation evolves. We have recently added several new features to the changelog, such as the [changelog diff](https://docs.bump.sh/product-updates/2023/11/02/changelog-diff/) to track for example the differences between two changes.
+
+Today, we are adding a new feature to the changelog: individual pages for each change in your documentation. Each release of your documentation generates a new entry in the changelog that now has its own page and unique URL. This allows you to easily share a link directly pointing to specific changes with members of your organization or your ecosystem.
```
Comment (scharrier) [inline src/_product-updates/2024-05-21-single-change.md:11]: I think we can add a small reminder here that they can add markdown content to this page to present/explain the API change, or write a full announcement.

File: src/_product-updates/2024-05-21-single-change.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Single Change
+tags: [New]
+image: /images/changelog/single-change.png
+---
+
+![single-change.png](/images/changelog/single-change.png)
+
+The changelog allows the consumers of your API documentation to track changes as your documentation evolves. We have recently added several new features to the changelog, such as the [changelog diff](https://docs.bump.sh/product-updates/2023/11/02/changelog-diff/) to track for example the differences between two changes.
```
Comment (scharrier) [inline src/_product-updates/2024-05-21-single-change.md:9]: There are 2 "to the changelog" (one in this paragraph, one in the following one). Maybe we can rephrase it a bit?

**Review by scharrier — COMMENTED** [review]
I left 2 small comments, and let @grossyoan review/approve it 👍

**Review by grossyoan — COMMENTED** [review]
Hi!

I'm not sure about the titles (both the H1 and the title of the cover image). We might want to have something easier to understand?

Maybe something like "Single change sharing"? Wdyt @jayk-u ?

File: src/_product-updates/2024-05-21-single-change.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Single Change
```
Comment (scharrier) [inline src/_product-updates/2024-05-21-single-change.md:2]:
```suggestion
title: Easily share your API changes
```

**Pull request closed without merging**